sites/docs/pages/components/link-button.md

```diff
@@ -3,25 +3,20 @@ title: Link Button
 sidebar_position: 1
 ---
 
-<LinkButton url='/components/link-button'>
+<LinkButton link='/components/link-button'>
   My Link Button
 </LinkButton>
 
 ```markdown
-<LinkButton url='/components/link-button'>
+<LinkButton link='/components/link-button'>
   My Link Button
 </LinkButton>
 ```
 
 ## Options
 
 
-<PropListing
-  name="url"
-  required
-  options="string"
->
-
-The URL to use for the link. Can be a full external link like `https://google.com` or an internal link like `/sales/performance`
+<PropListing name="link" required options='string'>
 
+Used to direct the page to a specified URL. It can accept either a full external link, such as `https://google.com`, or an internal link, such as `/sales/performance`
```

Comment:
```suggestion
Renders a button that, when clicked, navigates to the specified URL. It can accept either a full external link (e.g. `https://google.com`) or link to another page within your evidence project (e.g. `/sales/performance`).
```